src/main/pages/setup-kubernetes/kubernetes-admin-guide.md

```diff
@@ -0,0 +1,206 @@
+
+**What's CHE_INFRA_KUBERNETES_PVC_PRECREATE__SUBPATHS?**
+
+Pre 1.6 Kubernetes created subpaths within a PV with invalid permissions, sot hat a user in a running container was unable to write to mounted directories. When `CHE_INFRA_KUBERNETES_PVC_PRECREATE__SUBPATHS` is `true`, and a common strategy is used, a special pod is started before workspace pod is schedules, to pre-create subpaths in PV with the right permissions. You don't need to set it to true if you have Kubernetes 1.6+.
+
+**Restrictions**
+
+When a common strategy is used, and a workspace PVC access mode is RWO, only one Kubernetes node can simultaneously use PVC. You're fine if your Kubernetes/OpenShift cluster has just one node. If there are several nodes, a common strategy can still be used, but in this case, workspace PVC access mode should be RWM, ie multiple nodes should be able to use this PVC simultaneously (in fact, you may sometimes have some luck and all workspaces will be scheduled on the same node). You can change access mode for workspace PVCs by passing environment variable `CHE_INFRA_KUBERNETES_PVC_ACCESS_MODE=ReadWriteMany` to che deployment either when initially deploying Che or through che deployment update.
+
+Another restriction is that only pods in the same namespace can use the same PVC, thus, `CHE_INFRA_KUBERNETES_PROJECT` env variable should not be empty - it should be either Che server namespace (in this case objects can be created with che SA) or a dedicated namespace (token or username/password need to be used).
+
+## Unique PVC strategy
+
+It is a default PVC strategy, i.e. `CHE_INFRA_KUBERNETES_PVC_STRATEGY` is set to `unique`. Every workspace gets its own PVC, which means a workspace PVC is created when a workspace starts for the first time. Workspace PVC is deleted when a corresponding workspace is deleted.
+
+
+## Update
+
+An update implies updating Che deployment with new image tags. There are multiple ways to update a deployment:
+
+* `kubeclt edit dc/che` - and just manually change image tag used in the deployment
+* manually in OpenShift web console > deployments > edit yaml > image:tag
+* `kubectl set image dc/che che=eclipse/che-server:${VERSION} --source=docker`
+
+Config change will trigger a new deployment. In most cases, using older Keycloak and Postgres images is OK, since changes to those are very rare. However, you may update Keycloak and Postgres deployments:
+
+* eclipse/che-keycloak
+* eclipse/che-postgres
+
+You can get the list of available versions at [Che GitHub page](https://github.com/eclipse/che/tags).
+
+Since `nightly` is the default tag used in Che deployment, and image pull policy is set to Always, triggering a new deployment, will pull a newer image, if available.
```

Comment:
probably add link on how to change the image pull policy

Reply:
Done.

As to link to K8S admin guide, it is references from OpenShift Config page.

Reply:
![image](https://user-images.githubusercontent.com/5337267/38725902-60053706-3f11-11e8-9743-232b4b978a44.png)